.github/workflows/format.yml

```diff
@@ -7,6 +7,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
```

Comment:
Why 2 but not more?

Reply:
I need to use `HEAD^`, so I followed this guide https://github.com/actions/checkout#checkout-head